src/main/java/com/dpgil/pathlinker/path_linker/internal/PathLinkerPanel.java

```diff
@@ -629,11 +649,11 @@ private void initializeHiddenEdges() {
 
 		// only if we don't allow sources and targets internal to paths
 		if (!_allowSourcesTargetsInPaths) {
-			// hides all incoming edges to source nodes
+			// hides all incoming directed edges to source nodes
```

Comment:
OK. Explain this point in the comment.  When includeUndirected is true and allowSourcesTargetsInPaths is false, we should not hide any undirected edge incident on sources and targets since each such edge has a directed version leaving a source or entering a target.